doc/sphinx/source/frameworks/tensorflow_connect.rst

```diff
@@ -1,14 +1,134 @@
 .. frameworks/tensorflow_connect.rst:
 
-Connect TensorFlow\*
-====================
+Getting Started with TensorFlow\*
+=================================
 
 
-See the `README`_ on the `ngraph_bridge repo`_ for the many ways to connect 
-Tensorflow to nGraph, enabling a `DSO`_ backend that can speed up your TensorFlow 
-training and inference workloads.
+No matter what your level of experience with :abbr:`Deep Learning (DL)` systems 
+may be, nGraph provides a path to start working with the DL stack. Let's begin 
+with the easiest and most straightforward options.
 
+Using ``pip`` install
+----------------------
 
-.. _README: https://github.com/tensorflow/ngraph-bridge/blob/master/README.md
-.. _ngraph_bridge repo: https://github.com/tensorflow/ngraph-bridge  
-.. _DSO: http://csweb.cs.wfu.edu/%7Etorgerse/Kokua/More_SGI/007-2360-010/sgi_html/ch03.html
+If you are already familiar with TensorFlow\*, the easiest way to get started 
+is to use the latest `prebuilt nGraph-TensorFlow bridge`_. You can install 
+TensorFlow and nGraph to a virtual environment; otherwise, the code will install 
+to a system location.
+
+.. important:: The latest version of TensorFlow may be greater than the version 
+   named below.
+
+.. code-block:: console
+   
+   pip install -U tensorflow==1.14.1
```

Comment:
@avijit-nervana 

`pip install -U tensorflow==1.14.1` and `pip install -U ngraph-tensorflow-bridge` might not work right?
for the pypi package of the bridge tf1.13 is the correct version, right?